### DISCUSSION TITLE
Resources to consider

### DISCUSSION THREAD

**Discussion by avidrucker** [knowledge-for-good/knowledgeforgood#3] [Ideas]
Please suggest or ask about resources to incorporate into KFG

**Comment by avidrucker** [comment]
Clojure Overviews

**Reply by avidrucker** [reply]
https://www.creativeapplications.net/tutorials/introduction-to-clojure-part-1/

**Reply by avidrucker** [reply]
https://clojureverse.org/t/dram-is-an-interactive-beginners-clojure-guide-in-the-making/7268

**Reply by avidrucker** [reply]
https://github.com/kana-sama/awesome-clojure

**Reply by seekanddefine** [reply]
Why Learn Clojure

https://www.quora.com/Why-would-someone-learn-Clojure

Interesting responses <30

**Reply by seekanddefine** [reply]
`[How to Parse a Clojure Expression](https://mathematica.stackexchange.com/questions/111296/how-to-parse-a-clojure-expression)`

<30 min   Cool tree visual

**Comment by avidrucker** [comment]
How to write effective, concise, academic content for video

**Reply by avidrucker** [reply]
~~@seekanddefine I'd like to ask you for your help to analyze a video by this creator, such as this video: https://youtu.be/vKJpN5FAeF4~~ Converting to issue

**Reply by seekanddefine** [reply]
#57

**Comment by avidrucker** [comment]
Datomic / Datalog Overviews

**Reply by avidrucker** [reply]
https://www.youtube.com/watch?v=jPt4MiHFGy8

**Comment by avidrucker** [comment]
Team structure / topology

**Reply by avidrucker** [reply]
- [ ] Avi reads this article in full: https://www.instapaper.com/read/1421107333

**Comment by avidrucker** [comment]
Effective study techniques

**Reply by avidrucker** [reply]
https://psychology.ucsd.edu/undergraduate-program/undergraduate-resources/academic-writing-resources/effective-studying/index.html

**Reply by avidrucker** [reply]
https://www.edutopia.org/article/5-research-backed-studying-techniques

**Reply by avidrucker** [reply]
This looks dubious to me (use of "scientifically proven" w/o evidence, no cited author on article), but it does have links to dig into the sources: https://www.thebestcolleges.org/17-scientifically-proven-ways-to-study-better-this-year/

**Reply by avidrucker** [reply]
https://learningcenter.unc.edu/tips-and-tools/studying-101-study-smarter-not-harder/

**Reply by avidrucker** [reply]
https://www.csc.edu/learningcenter/study/studymethods.csc

**Reply by avidrucker** [reply]
https://www.mydegreeguide.com/how-to-study-tips/

**Reply by avidrucker** [reply]
https://www.usa.edu/blog/study-techniques/

**Comment by seekanddefine** [comment]
Tech Cooperatives

**Reply by seekanddefine** [reply]
The For-Purpose Enterprise: Holacracy, self-organization, fully distributed authority, powershift network and support for startups: https://encode.org/

**Reply by avidrucker** [reply]
> The For-Purpose Enterprise: Holacracy, self-organization, fully distributed authority, powershift network and support for startups: https://encode.org/

~~I read into Holocracy and it seems like it has a lot to offer. I'm curious to learn more, and didn't see much in the way in-depth, concrete examples. Do you know if it require some sort of buy in to get more info @seekanddefine ?~~ Converting to issue

**Comment by avidrucker** [comment]
Map / Atlas / Graph Formats

**Reply by avidrucker** [reply]
http://hyperphysics.phy-astr.gsu.edu/hbase/hframe.html

HyperPhysics is (IMO) a beautiful and technologically simple way to implement a knowledge map / concept map, encoded as hypermedia, similar to the hypermedia poetry you shared with me @seekanddefine

**Reply by avidrucker** [reply]
This seems like it has a similar heart & goal to many other great projects - to make data accessible. http://clojureatlas.com/

**Comment by avidrucker** [comment]
Ethical Systems Design

**Reply by avidrucker** [reply]
I believe this read strikes at the heart of automata, and some reasons why care is necessary: https://theartofresearch.org/trained-trust-a-two-way-refereed-feedback-mechanism-for-on-demand-work/

**Comment by seekanddefine** [comment]
- [ ] @seekanddefine and @avidrucker add time context to all suggest resources. (Ex. <2 min, <30 min).

**Comment by seekanddefine** [comment]
Lisp

**Reply by seekanddefine** [reply]
http://www.paulgraham.com/diff.html

Lisp embodied nine new ideas when it came out. 
Time: 5 min read

**Reply by seekanddefine** [reply]
Clojure is a Lisp, so all of the advantages that apply to Lisps apply to Clojure. Here's a good essay by Paul Graham describing some of the core values of Lisps: http://www.paulgraham.com/diff.html (for a good discussion of this essay, see: http://stackoverflow.com/questions/2710231/please-explain-some-of-paul-grahams-points-on-lisp/2710450)

**Reply by seekanddefine** [reply]
https://gist.github.com/rduplain/c474a80d173e6ae78980b91bc92f43d1

Rich Hickey's article and other resources on the article >30min

**Reply by seekanddefine** [reply]
[Clojure Bowling Parser](https://blog.taylorwood.io/2018/05/17/instaparse.html)
<30min
